apps/mull-ui-e2e/src/integration/US-3.5.spec.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference types="Cypress" />
+import 'cypress-file-upload';
+import { frameSizes } from './../fixtures/frame-sizes';
+
+frameSizes.forEach((frame) => {
+  describe(`US-3.5: Group Chat Page (${frame.name} view)`, () => {
+    beforeEach(() => {
+      cy.interceptGraphQLReqs();
```

Comment:
```suggestion
```
This E2E was failing for me due to this line. You can just remove this since you are not using it anyway (it is used in a wait clause, example [here](https://github.com/RGPosadas/Mull/blob/develop/apps/mull-ui-e2e/src/integration/US-7.2.spec.ts#L26)). In fact, US-3.1 and 3.2 don't use it either so you can remove it from there as well.

Reply:
let me know if it works. I still get test on develop that fails on my end

Reply:
Yeah so I had already tested this for US-3.1, 3.2, and 3.5 E2Es and they all still pass without that line, so please don't forget to remove it before merging

Reply:
I will remove it only for this e2e test unless told otherwise